data_processing/fixed_cost_assumptions.md

```diff
@@ -0,0 +1,55 @@
+## Scenario 11: Addition of fixed costs
```

Comment:
You should add actual links and citations for the data.

Reply:
What format for citations/links do you want?

Reply:
See Roberto's markdown documents. Typically "here is a quote" [1]


References:
[1] [a link](thelink)